edkrepo/commands/clone_command.py

```diff
@@ -181,5 +182,5 @@ def run_command(self, args, config):
             # Command line disables sparse checkout
             use_sparse = False
         if use_sparse:
-            print(SPARSE_CHECKOUT)
+            ui_functions.print_info_msg(SPARSE_CHECKOUT)
             sparse_checkout(workspace_dir, repo_sources_to_clone, manifest)

```

Comment:
From the screenshots you have posted I think that the sparse_checkout() function may also have prints can you update these as part of this review as well so that all prints during a clone are consistent.